examples/features/webhook-smartvf/README.md

```diff
@@ -0,0 +1,150 @@
+# Client requests for postgresql service over SmartVF
```

Comment:
Can we move this example into ovs folder?

Reply:
I do not think the ovs folder is suited for this example. The reason is that in this example we showcase the ability to inject interface pool resources through admission webhook  by adding the related sriovToken label to the annotation. The forwarder that you are using do not play any role. I have just chose to use the ovs forwarder to showcase the aforementioned ability. I could have used sriov forwarder as well. So I think we should not move the example under any forwarder (ovs or sriov).

Reply:
note: features suite is using the setup with forwawrder-vpp. I am not sure that this forwarder will correctly work on non-sriov clusters.

Reply:
This example showcases the capability to inject SRIOV or SMARTVF interface resources by using the sriovToken Label in the annotation nsm section. Is not related to the forwarder at all. I just chose the ovs forwarder to deploy the example. I could just use sriov-forarder or vpp-fowarder (If vpp had support to create NSM connections through SRIOV interfaces) as well.

note: ovs-forwarder works equally well in non-sriov clusters. It uses veth interfaces to realize the nsm connections.